adafruit_74hc595.py

```diff
@@ -32,6 +32,7 @@
     import typing  # pylint: disable=unused-import
     from microcontroller import Pin
     import busio
+    from circuitpython_typing import ReadableBuffer
```

Comment:
I think it's explicitly pip installed by `adafruit-circuitpython-busdevice`, but we should also add this library to `requirements.txt` explicitly.  Can you add `adafruit-circuitpython-typing` to `requirements.txt`?